tests/WebAssertTest.php

```diff
@@ -982,6 +982,81 @@ public function testElementAttributeExists()
         );
     }
 
+    public function testElementAttributeNotExistsValid()
```

Comment:
I've expected, that you'll just copy-paste (and make changes) test for `elementAttributeExists` method.